content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -228,6 +228,15 @@ and Ready or completely terminated prior to launching or terminating another
 Pod. This option only affects the behavior for scaling operations. Updates are not
 affected.
 
+#### Minimum Ready Seconds
+
+{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+
```

Comment:
Let's explain that this field only works if you enable the `StatefulSetMinReadySeconds` feature gate.

Reply:
Sure, I added it [here](https://github.com/kubernetes/website/pull/27683/files#diff-ff635592a5e94bba6c6677379e9a5d64b4a035846dfe25881c1f05671e39b8bdR240). PTAL.